hedera-node/hedera-network-admin-service-impl/src/main/java/com/hedera/node/app/service/networkadmin/impl/utils/NetworkAdminServiceUtil.java

```diff
@@ -0,0 +1,47 @@
+/*
+ * Copyright (C) 2023 Hedera Hashgraph, LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.hedera.node.app.service.networkadmin.impl.utils;
+
+import static java.lang.System.arraycopy;
+
+import com.google.common.primitives.Longs;
+import com.hedera.hapi.node.base.AccountID;
+import com.swirlds.common.utility.CommonUtils;
+import edu.umd.cs.findbugs.annotations.NonNull;
+
+/**
+ * Provides utility methods for network admin operations.
+ */
+public class NetworkAdminServiceUtil {
```

Comment:
Util class should be `final` with private constructor throwing an exception.

```suggestion
public final class NetworkAdminServiceUtil {
```